pkg/operator/cluster/controller.go

```diff
@@ -144,8 +144,8 @@ func (c *ClusterController) onAdd(obj interface{}) {
 	}
 
 	if cluster.Spec.MonCount <= 0 {
-		logger.Warningf("mon count is 0 or less (given: %d), should be greater than 0, defaulting to %d", cluster.Spec.MonCount, defaultMonCount)
-		cluster.Spec.MonCount = defaultMonCount
+		logger.Warningf("mon count is 0 or less (given: %d), should be greater than 0, defaulting to uneven count of nodes (at maximum %d)", cluster.Spec.MonCount, defaultMonCount)
```

Comment:
what if we just log that we will use the default number of mons (but since it's dynamic, let's not print the number)